hypothesizer/requirements.txt

```diff
@@ -0,0 +1,9 @@
+click==6.7
```

Comment:
idk if you care but there's a buncha junk in here that your app doesn't need, probs just hangin' around in your own python... i assume you did `pip freeze`?

Reply:
nnnnot that i know of, except to do everything in virtualenvs and keep them all really clean? :(